### ISSUE TITLE
getmail4?

### ISSUE THREAD

**Issue by roswitina** [mailcow/mailcow-dockerized#452]
Is there a way to integrate getmail4?
thx
rosi

**Comment by andryyy** [comment]
Hi, we use imapsync now. :-)

**Comment by roswitina** [comment]
Excuse me; Could have read better before.
Rosi

**Comment by roswitina** [comment]
One thing is still not clear to me. How can I pick one or more pop3 accounts?
Rosi

PS.: Ich stehe auf der Leitung..... ?!?!?

**Comment by andryyy** [comment]
Hi,

That's not available. Only IMAP can be synced.

**Comment by Braintelligence** [comment]
Do you really have no Imap connection to your old mail server?

**Comment by roswitina** [comment]
I want to use mailcow in my intranet for collecting my mails. Since I work with several devices, I want an internal mailserver. I want to retrieve the mails from the various providers from the Mailserver. Currently I use an old Zentyal 3.4, which collects the mails. I came across by chance on "Mailcow" and he likes me a lot. Therefore, I wanted to switch to this current product. I want to use mailcow in my intranet for collecting my mails. Since I work with several devices, I want an internal mailserver. I want to retrieve the mails from the various providers from the Mailserver. Currently I use an old Zentyal 3.4, which collects the mails. I came across by chance on "Mailcow" and he likes me a lot. Therefore, I wanted to switch to this current product. With getmail4 (mailcow 0.14) it works so synonymous. I will try again imap in any case.

Thank you
Rosi
